### ISSUE TITLE
[a11y] 9.1.4.13 - Faded in content is operable

### ISSUE THREAD

**Issue by tbsbdr** [owncloud/web#5387]
- Context: The Web UI should be accessible according to the WCAG2.1 /BITV 2.0 guidelines. The number in the title relates to the test-steps: https://www.bitvtest.de/bitv_test/das_testverfahren_im_detail/pruefschritte.html
- "Status" relates to the evaluation scheme https://www.bitvtest.de/bitv_test/das_testverfahren_im_detail/verfahren.html#c1262 
- "a11y" is short for accessibility
- browser zoom has been set to 200% as of https://github.com/owncloud/web/issues/5385
- ----------------

## General notes
When moving the mouse pointer over several buttons e.g."Personen hinzufügen" or "Bearbeite öffentlichen Link" user defined tooltips are appearing. It is not possible for users to move the mouse pointer within these tooltips without making them disappear. Because the content is often only partially visible due to the heavy zoom factor, users must be able to move the pointer from the triggering element over the displayed content.

## Status: partly fulfilled

**Comment by pascalwengerter** [comment]
Perhaps can easily be fixed by configuring tippys to be interactive (https://atomiks.github.io/tippyjs/#interactivity), investigating this now
